index.html

```diff
@@ -60,14 +60,14 @@ <h2 class="soft-dev">I am a Software Developer</h2>
           <img
             class="item-icon"
             src="img/social-icons/light-icons/Vector-1.png"
-            alt="Twitter Logo"
+            alt=""
```

Comment:
- [ ] It looks like you forgot to add alt text here, please kindly add it.

Reply:
I am going to add it. Thanks